spec/transaction.test.js

```diff
@@ -57,7 +54,7 @@ describe("Transaction", () => {
   });
 
   describe("#formatTransaction", () => {
-    it("formats the credit transaction into a string", () => {
+    it("formats the credit and debit transaction into a string", () => {
```

Comment:
nice test! I think it might be worth breaking it out into two though as you've got multiple expectations in this one

Reply:
will do. Thank you!